### DISCUSSION TITLE
Is there a way to quickly build a list of named aliases?

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#1824] [Help]
I know that I can select _Alias_ as a thing to search on. I'm looking for something more. Sort of like the way one can build a table of contents in Word. Where I could build a list of aliases that are linked to the proper nodes.  I can obviously do this manually, but take a look at my mindmap, unfolded to just a few levels. This map has been developed since the end of 2021 and has more than 21,000 nodes. Building the list of aliases manually is not going to happen. Is there a way to generate such a list? Perhaps a script?

Thanks in advance.
~ Bal

![20000nodemindmap](https://github.com/freeplane/freeplane/assets/8197883/846f0623-03fb-4500-91a1-f4e027cd5d33)

**Comment by macmarrum** [comment]
You can try [createAliasesBranch.groovy](https://raw.githubusercontent.com/macmarrum/freeplane-scripting/main/src/scripts/createAliasesBranch.groovy) -- as usual, you can adjust the menu location by changing `ON_SINGLE_NODE="/menu_bar/Mac1"`
When executed, the script will create a new branch with all your aliases. 
A star icon means a global node, a key icon means an encrypted node.
Each alias has, as its child, a clone of the original node or the node's copy, if the original node is encrypted.

**Comment by wordmuse** [comment]
Perfect - thank you very much - this has made it much easier to see landmarks in my map and then either keep them, alter or delete them. Really helpful. 😊

~ Bal
